test/cmdlineTests.sh

```diff
@@ -133,37 +151,44 @@ function test_solc_behaviour()
     if [[ $exitCode -ne "$exit_code_expected" ]]
     then
         printError "Incorrect exit code. Expected $exit_code_expected but got $exitCode."
-        rm -f $stdout_path $stderr_path
         exit 1
     fi
 
     if [[ "$(cat $stdout_path)" != "${stdout_expected}" ]]
     then
         printError "Incorrect output on stdout received. Expected:"
-        echo -e "${stdout_expected}"
```

Comment:
I'm not sure if this `-e` have any purpose so I removed it.

Reply:
I think it's fine to remove it. I also don't see a need for that at the moment.